frontend/src/components/Timer.tsx

```diff
@@ -0,0 +1,58 @@
+import Countdown from "react-countdown";
+import { Text, keyframes } from "@chakra-ui/react";
+import { useState } from "react";
+
+interface TimerThings {
```

Comment:
Same here if you choose to rename the other type

Reply:
Addressed!